### PR TITLE
Remove unused files

### PR DESCRIPTION
**Pull request by McModknower**
# Description

There was some unused code/files that nobody needs

## Type of change
Just put an x in the [] which are valid.
- [x] Code cleanup (non-breaking change which makes working with the code easier or similar)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: app/src/main/java/swati4star/createpdf/util/CreatePdf.java [hunk 1]
```diff
@@ -144,11 +142,6 @@ protected String doInBackground(String... params) {
 
                 Log.v("Stage 5", "Image compressed " + qualityMod);
 
-                BitmapFactory.Options bmOptions = new BitmapFactory.Options();
-                Bitmap bitmap = BitmapFactory.decodeFile(mImagesUri.get(i), bmOptions);
-
-                Log.v("Stage 6", "Image path adding");
-
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/util/CreatePdf.java:151]: are you sure we don't need this section?

Comment (McModknower) [reply app/src/main/java/swati4star/createpdf/util/CreatePdf.java:151]: Android studio showed me the bitmap variable wasn't used and after deleting that the bmOptions variable also was marked as not used

Comment (Swati4star) [reply app/src/main/java/swati4star/createpdf/util/CreatePdf.java:151]: I understand. But, I remember having some issues with functionality after removing this section. 

Can we leave it as is?
The removal of other files is good

Comment (McModknower) [reply app/src/main/java/swati4star/createpdf/util/CreatePdf.java:151]: ok

**Pull request merged**